docs/operators/ftpInputOperator.md

```diff
@@ -0,0 +1,54 @@
+FTP Input Operator
+=============
+
+## Operator Objective
+This operator is designed to scan a directory from an FTP server for files, read
```

Comment:
Mention the class name here, e.g. This operator (`AbstractFTPInputOperator`) ...